Install lib*-types.conf

This patch was written by Jérôme Pouiller and has been already sent to
upstream:

http://lists.alioth.debian.org/pipermail/ltrace-devel/2015-June/001328.html

Signed-off-by: Vicente Olivert Riera <Vincent.Riera@imgtec.com>

Install lib*-types.conf

Files etc/libpthread.so-types.conf and etc/libc.so-types.conf was introduced in
commit 5ba9e10 ("Split type definitions from the bundled configs into their own
files"). However, they was not installed.

So latrce failed with messages like :
/usr/share/ltrace/libm.so.conf:333: error: unknown type around 'ldouble erfl(ldouble);

This patch declare these configuration files.

Signed-off-by: Jérôme Pouiller <jezz at sysmic.org>
---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index f63fa09..394d91c 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -72,7 +72,8 @@ dist_man5_MANS = ltrace.conf.5
 dist_doc_DATA = COPYING CREDITS INSTALL README TODO
 
 dist_pkgdata_DATA = etc/syscalls.conf etc/libc.so.conf	\
-	etc/libm.so.conf etc/libacl.so.conf etc/libpthread.so.conf
+	etc/libm.so.conf etc/libacl.so.conf etc/libpthread.so.conf \
+	etc/libpthread.so-types.conf etc/libc.so-types.conf
 
 EXTRA_DIST = \
 	debian/changelog \
-- 
1.8.5.6
